themes/default/content/docs/intro/pulumi-service/accounts.md

```diff
@@ -21,13 +21,13 @@ up. You may use any of the following identities to sign up:
 * [Email](https://app.pulumi.com/signin/email)
 * [Single Sign-on](https://app.pulumi.com/signin/sso)
 
-Your account lets you authenticate into the Pulumi Console, where you can do the
+Your account lets you authenticate in to the Pulumi Service, where you can do the
```

Comment:
Since [the MS Style Guide's discussion of `sign in`](https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/s/sign-in-sign-out) refers to the verb form as two words and as we could replace those two words with `authenticate`, I'd not use `in` here. When we're authenticating, we're authenticating to a service:
```suggestion
Your account lets you authenticate to the Pulumi Service, where you can do the
```